.circleci/main.yml

```diff
@@ -1468,6 +1468,7 @@ jobs:
             sleep 20
             kubectl wait -A --for=condition=ready pod -l app=pachd --timeout=5m
             until poetry run python -c "import pachyderm_sdk; pachyderm_sdk.Client(\"$(minikube ip)\", 30080).get_version()"; do sleep 5; done
+            until poetry run python -c "import pachyderm_sdk; pachyderm_sdk.Client(\"$(minikube ip)\", 30081).get_version()"; do sleep 5; done
```

Comment:
You should bring this up-to-date with current master. I shouldn't need this change